test-lit/Semantics/common.sh

```diff
@@ -11,18 +11,12 @@ function die {
   exit 1
 }
 
-case $# in
-  (1) ;;
-  (2) F18=$2 ;;
-  (*) echo "Usage: $(basename $0) <fortran-source> [<f18-executable>]"; exit 1
-esac
-[[ -z ${F18+x} ]] && die "Path to f18 must be second argument or in F18 environment variable"
-[[ ! -f $F18 ]] && die "f18 executable not found: $F18"
 case $1 in
   (/*) src=$1 ;;
   (*) src=$(dirname $0)/$1 ;;
 esac
 USER_OPTIONS=`sed -n 's/^ *! *OPTIONS: *//p' $src`
 echo $USER_OPTIONS
-temp=`mktemp -d ./tmp.XXXXXX`
-[[ $KEEP ]] || trap "rm -rf $temp" EXIT
+F18=$2
+temp=$3
+mkdir -p $temp
```

Comment:
This used to report an error if the wrong number of arguments is passed in. That seems like a good thing.

Reply:
The wrong number of arguments can't be passed in (so it's not needed) as far as the automated tests are concerned, I'm sure people manually using the tests will know what they're doing (but they shouldn't since new tests should be gtest or lit), and the main reason I took this out was to be able to add %t as an argument, and I couldn't otherwise.